util/systemtime/src/lib.rs

```diff
@@ -4,7 +4,10 @@ mod test_realtime;
 
 #[cfg(feature = "enable_faketime")]
 use std::sync::atomic::{AtomicBool, AtomicU64, Ordering};
-use std::time::Duration;
+#[cfg(not(target_family = "wasm"))]
+pub use std::time::{Duration, Instant, SystemTime};
+#[cfg(all(target_family = "wasm", target_os = "unknown"))]
```

Comment:
Why `target_os = "unknown"`.

Reply:
`web-time` doesn't support `wasm32-wasi*`, it only support `wasm32-unknown-*` on browser